app/resources/api/v2/request_metadata_resource.rb

```diff
@@ -28,9 +28,9 @@ class RequestMetadataResource < BaseResource
       # Attributes
       ###
 
-      # @!attribute [r] number_of_samples_per_pool
-      #   @return [Int] the number_of_samples_per_pool.
-      attribute :number_of_samples_per_pool, write_once: true
+      # @!attribute [r] number_of_pools
+      #   @return [Int] the number_of_pools.
```

Comment:
Bit more detail?

```suggestion
      #   @return [Int] the number_of_pools requested in the Submission. As used in the scRNA Core pipeline, it is specified at the Study-Project level: it will have the same value for all Requests that share the same Study and Project. It is used in the pooling algorithm.
```

Reply:
Thanks for the suggestion. I have now updated the comments of both `number_of_pools` and `cells_per_chip_well` attributes.